addons/bus/static/tests/mock_websocket.js

```diff
@@ -76,6 +77,10 @@ export function patchWebsocketWorkerWithCleanup(params = {}) {
     websocketWorker = websocketWorker || new WebsocketWorker();
     websocketWorker.INITIAL_RECONNECT_DELAY = 0;
     websocketWorker.RECONNECT_JITTER = 0;
+    websocketWorker._debouncedUpdateChannels = debounce(
+        websocketWorker._updateChannels.bind(websocketWorker),
+        0
+    );
```

Comment:
Do we need this? I don't like that tests don't use the real thing.

Reply:
Every test waiting for bus subscription will take 500ms if we don't do this which is a lot.